### PR TITLE
[ENH] support Serepent 2.2.1

### LINKED ISSUE #480: ENH Support Serpent 2.2.1
**Is your feature request related to a problem? Please describe.**
Serpent2.2.1 has been released, but it's not included in [options](https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/b53d14e1ed9d6f8090218851f675144cb6696a30/serpentTools/settings.py#LL109C12-L109C12)

**Describe the solution you'd like**
I think repeating the same procedure as in #467 should be enough, i.e.
- [x] check if there are any changes in `_res.m` files in 2.2.1
- [x] `2.2.1` is added to [serpentVersion](https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/b53d14e1ed9d6f8090218851f675144cb6696a30/serpentTools/settings.py#LL109C12-L109C12) 

**Describe alternatives you've considered**
I think it's good to have support for all versions currently in use.
I guess this should be pretty straightforward.

**Additional context**
(None)

### PR DESCRIPTION
**Pull request by rzehumat**
Closes #480 

Make sure you have read over the developer guide to ease
the review process. These include:

- [x] PR fits the [project scope](http://serpent-tools.readthedocs.io/en/latest/develop/contributing.html#project-scope)
- [x] Code to be committed is [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and matches the [code style](http://serpent-tools.readthedocs.io/en/latest/develop/codestyle.html#code-style) of the project
- [x] New and/or changed features are [well documented](http://serpent-tools.readthedocs.io/en/latest/develop/documentation.html#documentation) and have adequate testing
- [x] For first-time contributors, you have included your attribution in [`docs/contributors.rst`](https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/develop/docs/contributors.rst)

Include a thorough and concise overview of the changes, and why this PR is overall beneficial to the project.

Once the PR has been created and is nearing closure, make sure that serious changes, including deprecations and incompatible changes are documented in [the changelog](https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/develop/docs/changelog.rst)

### CODE REVIEW

**Comment by rzehumat** [comment]
Check `MapStrVersions`
- [ ] meta
- [ ] rslt
- [ ] univ
- [ ] days
- [ ] burnup
- [ ] infxs
- [ ] b1xs
- [ ] varsUnc

**Comment by rzehumat** [comment]
Ok, I think it's ready for review, @drewejohnson 

- It only adds `2.2.1` to supported versions.
  -  I've quickly checked with the changelog and 1 `_res.m` file... If more testing is needed, let me know. 
  - This changed only 2 files
- The remaining ~29 changed files result from running [codespell](https://github.com/codespell-project/codespell) to check for typos

File: serpentTools/parsers/base.py [hunk 1]
```diff
@@ -119,7 +119,7 @@ class SparseReaderMixin(object):
     ``sparse`` passed to the constructor and if :term:`scipy` is
     installed.
 
-    Paramters
+    Parameters
     ---------
```
Comment (drewejohnson) [inline serpentTools/parsers/base.py:123]: I think we'll need an extra `-` to get the docstring to render properly after this change
```suggestion
    ----------
```

**Review by drewejohnson — CHANGES_REQUESTED** [review]
Thanks for the contribution @rzehumat! Welcome to the repo

The changes look good to me. One minor change and we'll be good to go

File: serpentTools/parsers/base.py (hunk 1 above)
Comment (rzehumat) [reply serpentTools/parsers/base.py]: Good point, thanks

**Comment by rzehumat** [comment]
Now I'm unsure why I see "Changes requested" here. I thought your change request is resolved.

**Comment by drewejohnson** [comment]
> Now I'm unsure why I see "Changes requested" here. I thought your change request is resolved.

It's just because the last review had changes requested. You applied them, and it looks great!

**Review by drewejohnson — APPROVED** [review]

**Pull request merged**